### PR TITLE
Guilded.Guilded version 1.0.996954

### PR DESCRIPTION
**Pull request by KaranKad**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/14725)

### CODE REVIEW

File: manifests/g/Guilded/Guilded/1.0.996954/Guilded.Guilded.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+PackageIdentifier: Guilded.Guilded
+PackageVersion: 1.0.996954
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
```
Comment (OfficialEsco) [inline manifests/g/Guilded/Guilded/1.0.996954/Guilded.Guilded.installer.yaml:12]:
```suggestion
  Architecture: neutral
```

File: manifests/g/Guilded/Guilded/1.0.996954/Guilded.Guilded.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate
+
+PackageIdentifier: Guilded.Guilded
+PackageVersion: 1.0.996954
```
Comment (OfficialEsco) [inline manifests/g/Guilded/Guilded/1.0.996954/Guilded.Guilded.installer.yaml:4]:
```suggestion
PackageVersion: 1.0.996954-release
```

File: manifests/g/Guilded/Guilded/1.0.996954/Guilded.Guilded.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate
+
+PackageIdentifier: Guilded.Guilded
+PackageVersion: 1.0.996954
+PackageLocale: en-US
+Publisher: Guilded
```
Comment (OfficialEsco) [inline manifests/g/Guilded/Guilded/1.0.996954/Guilded.Guilded.locale.en-US.yaml:6]:
```suggestion
Publisher: Guilded, Inc.
```

File: manifests/g/Guilded/Guilded/1.0.996954/Guilded.Guilded.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+PublisherUrl: https://www.guilded.gg/
+PublisherSupportUrl: https://support.guilded.gg/
+PrivacyUrl: https://support.guilded.gg/hc/en-us/articles/360039235054-Privacy
+PackageName: Guilded
+PackageUrl: https://www.guilded.gg/
+License: Copyright (c) Guilded
+ShortDescription: Guilded upgrades your group chat and equips your server with integrated event calendars, forums, and more – 100% free.
+Moniker: guilded
+Tags:
+- chat
```
Comment (OfficialEsco) [inline manifests/g/Guilded/Guilded/1.0.996954/Guilded.Guilded.locale.en-US.yaml:16]:
```suggestion
- discord
- chat
- voip
- voice
- voice-chat
- gaming
- guild
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Hmm, i think it needs to be like this to match ARP.
Are you going to remove #14672 when this is merged?

**Comment by KaranKad** [comment]
I'll create new pr

**Pull request closed without merging**